src/System.Net.Security/src/System/Net/_SslSessionsCache.cs

```diff
@@ -135,15 +135,15 @@ internal static SafeFreeCredentials TryCachedCredential(byte[] thumbPrint, SslPr
 
             SafeCredentialReference cached = s_CachedCreds[key] as SafeCredentialReference;
 
-            if (cached == null || cached.IsClosed || cached._Target.IsInvalid)
+            if (cached == null || cached.IsClosed || cached._target.IsInvalid)
```

Comment:
Nit: ._ should not be used. Since `_target` is internal it should be renamed to `Target`.